README.md

```diff
@@ -29,77 +29,145 @@ How to integrate the SDK
 * Define which Activity should be launched to request the user to log in 
```

Comment:
If you are wondering why I added the surrounding class and method to each method call sample code it's because using "```java" adds code validation and warns me that a class & method is expected. I like it anyway better like this as this helps the SDK user while implementing and the chances are higher that they implement it similar to us